app/views/procedures/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @procedures.each do |p|%>
+    <div><%=p.name%>
+    
+    <%p.facilities.each do |f|%>
+        <%=f.name%>
```

Comment:
This is an instance of the N+1 query problem, which can be helped by adding [eager loading](https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations).

Reply:
Deleted this page, will work on it from scratch later